gravitee-am-ui/src/app/domain/applications/application/advanced/general/general.component.html

```diff
@@ -132,6 +132,21 @@ <h5>Post logout redirect URIs</h5>
           </div>
         </div>
 
+        <div *ngIf="displaySection()" class="gv-form-section">
+          <div class="gv-form-section-title">
+            <h5>Single Sign Out</h5>
+            <mat-divider></mat-divider>
+          </div>
+          <div fxLayout="column">
+            <mat-slide-toggle
+              (change)="enableSingleSignOut($event)"
+              [checked]="this.singleSignOut" [disabled]="!editMode">
+              Logout from the external OIDC provider
+            </mat-slide-toggle>
+            <mat-hint style="font-size: 75%;">Call the end session endpoint of the external OIDC provider if available on AM logout call.</mat-hint>
```

Comment:
With SAML 2.0 it's not only OIDC provider. I think we can replace by external identity provider.